src/docfx/config/ops/CredentialHandler.cs

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+namespace Microsoft.Docs.Build
+{
+    internal class CredentialHandler : DelegatingHandler
```

Comment:
>CredentialHandler [](start = 19, length = 17)

shall we call this `OpsCredentialHandler`?

Reply:
it handles all the credentials not only ops